doc/Language/grammar_tutorial.pod6

```diff
@@ -189,21 +160,22 @@ that has all 3 parameters included:
     #  data => ｢7/notify｣
```

Comment:
If that's what it outputs in the console, then I'd keep it. I'm just wondering why they're there and why nothing else uses them. If they're just objects, why aren't they printed like other objects?